data/org.freedesktop.portal.WebExtensions.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0"?>
+<!--
+ Copyright (C) 2022 Canonical Ltd
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+  <!--
+      org.freedesktop.portal.WebExtensions:
+      @short_description: WebExtensions portal
+
+      The WebExtensions portal allows sandboxed web browsers to start
+      native messaging servers installed on the host system.
```

Comment:
(This applies to the whole patch)

The correct terminology is "native messaging host", not "native messaging server".

That name is documented at https://developer.chrome.com/docs/apps/nativeMessaging/

[nativeMessaging on MDN](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Native_messaging) currently refers to the binary as "native application", but that is not unambiguous in the portal, so I recommend to stick to "native messaging host".

Or if you'd like, describe "Native Messaging Host" once and refer to the application as "NMH app".